src/sources/MVT.js

```diff
@@ -81,6 +61,7 @@ const MVT_TO_CARTO_TYPES = {
  *
  * @api
  */
+import Worker from './MVTWorkers.worker';
```

Comment:
I would put the import below the rest of import declarations.

Reply:
:+1: